index.html

```diff
@@ -343,7 +354,7 @@ <h2 class="top-text1">About Myself</h2>
       </section>
 
       <!-- Form -->
-      <section id="contact-form" class="form-container">
+      <section id="contact" class="form-container">
         <div class="form-content-container">
           <!-- Form header -->
           <div class="form-header-container">
```

Comment:
- [ ] Great work on creating the contact form! However, as previously mentioned, it is important for your project to match the Figma template. Please make the following changes to ensure that the contact form is aligned with the template:

- The elements of the form, including the "Get in touch" button, should be centered on the page.
- Verify that the description is also centered.
- The background image needs to be distinct from the one used in the mobile version.

**Your design**

![Screenshot from 2023-03-29 08-30-49](https://user-images.githubusercontent.com/105117832/228554441-65484a61-c1cc-4b71-a187-bc3bbf85d9e6.png)

**Figma**

![Screenshot from 2023-03-29 08-30-28](https://user-images.githubusercontent.com/105117832/228554768-d9575d43-0a85-481d-9f16-d516e7683dfd.png)

Reply:
I improved the Contact form.